public/app/controllers/objects_controller.rb

```diff
@@ -26,17 +26,18 @@ def index
       params[:limit] = 'digital_object,archival_object' unless params.fetch(:limit, nil)
       params[:op] = ['OR']
     end
-    page = Integer(params.fetch(:page, "1"))
     search_opts = default_search_opts(DEFAULT_OBJ_SEARCH_OPTS)
     search_opts['fq'] = ["repository:\"/repositories/#{repo_id}\""] if repo_id
     @base_search = repo_id ? "/repositories/#{repo_id}/objects?" : '/objects?'
 
+    search_opts['resolve[]'] = ['linked_instance_uris:id'] if params[:limit].include? 'digital_object'
```

Comment:
@donaldjosephsmith The problem isn't only for the Digital Materials pre-configured results list, the problem is search results in general. So checking only the `:limit` param here is insufficient, as it doesn't send the data when `:limit` is not part of the query.

So try for example, searching for all records (http://localhost:3001/search?q[]=*&op[]=&field[]=keyword&from_year[]=&to_year[]=), or searching for the name of a digital object with a linked instance.